src/screens/UserPortal/Events/Events.module.css

```diff
@@ -14,14 +14,9 @@
   max-width: 800px;
 }
 
-.colorLight {
-  background-color: #f1f3f6;
-}
-
```

Comment:
Why are you removing light color from all screens?

Reply:
because it is making all the screens look odd